apis/v1beta1/validation/httproute.go

```diff
@@ -334,6 +342,31 @@ func validateHTTPHeaderModifier(filter gatewayv1b1.HTTPHeaderFilter, path *field
 	return errs
 }
 
+func validateHTTPRouteTimeouts(timeouts *gatewayv1b1.HTTPRouteTimeouts, path *field.Path) field.ErrorList {
+	var errs field.ErrorList
+	errorString := fmt.Sprintf("timeout value must be greater than or equal to %s", timeoutMinValue)
+	if timeouts.BackendRequest != nil {
+		backendTimeout := timeouts.BackendRequest.Duration
+		if backendTimeout < timeoutMinValue && backendTimeout != 0 {
```

Comment:
As mentioned above, I think we should have a min and max value here and that 0 should be invalid.

Reply:
Resolving since we've allowed "0s" intentionally.